app/src/main/java/com/group80/uoftinder/chat/ContactModel.java

```diff
@@ -0,0 +1,101 @@
+package com.group80.uoftinder.chat;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * A model for Firebase Firestore, storing basic contact info
+ */
```

Comment:
I'm not entirely sure where this would be instantiated and when it would be created but to adhere to Clean Architecture, you may want to consider a ContactModelFactory to create ContactModel instances (similar to the Message Factory)

Reply:
Good idea! Not essential to function, but keeps the code clean.

Reply:
I'm not instantinating this class, at least not in this PR -- this belongs to another feature (see #61)